code/game/gamemodes/endgame/endgame.dm

```diff
@@ -9,21 +9,20 @@
 
  // Default shit.
 /datum/universal_state
-	// Just for reference, for now.
-	// Might eventually add an observatory job.
- 	var/name = "Normal"
- 	var/desc = "Nothing seems awry."
+	//Just for reference, for now.
+	//Might eventually add an observatory job.
+	var/name = "Normal"
+	var/desc = "Nothing seems awry."
 
- 	// Sets world.turf, replaces all turfs of type /turf/space.
- 	var/space_type         = /turf/space
```

Comment:
Okay removing the space before the comment is arguable (although I hate it), but removing alignment? What the hell is wrong with you.